core/ast/src/main/java/org/lgna/project/virtualmachine/VirtualMachine.java

```diff
@@ -1237,6 +1241,15 @@ public List<VirtualMachineListener> getVirtualMachineListeners() {
     }
   }
 
+  public void setForSceneEditor() {
+    isForRunning = false;
+  }
+
   private final List<VirtualMachineListener> virtualMachineListeners = Lists.newLinkedList();
   private boolean isStopped = false;
+
+  // Marks this VM for use in running worlds. When true it allows errors to be thrown that interrupt execution.
+  // A value of false indicates this VM is used during scene loading or scene setup where thrown exceptions can
+  // cause these processes to break and should be simply logged and the setup code continued.
+  private boolean isForRunning = true;
```

Comment:
nitpicky name question, should this variable maybe be named something like "allowExceptions" or "quietlyLogErrors"